packages/api/src/controllers/asset.ts

```diff
@@ -1208,6 +1208,39 @@ app.patch(
   }
 );
 
+app.post("/:id/retry", authorizer({ anyAdmin: true }), async (req, res) => {
```

Comment:
Either way, do these APIs need to be admin-only? Feels like it could be useful for users to retry their tasks if they want (dashboard integration would be a plus). Remember to add a check for the user Id if you remove the admin restriction.